apps/waterboilingpoint/README.md

```
@@ -0,0 +1,21 @@
+# Water Boiling Point for Tronbyt
+Displays the current boiling point for water at a given location.
+
+![Water Boiling Point for Tronbyt](water_boiling_point.webp)  
+*The boiling point of water in Denver, Colorado, USA on August 17, 2026.*
+
+## Setup
+This app requires an API key from OpenWeather. If you don't already have one, you can get one for free [here](https://home.openweathermap.org/users/sign_up).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)